### ISSUE TITLE
ElasticSearch data has wrong date

### ISSUE THREAD

**Issue by aaronrryan** [StamusNetworks/SELKS#55]
The data that is being pushed into ES has the wrong datetime.
I have set the system to PDT timezone, but it inserting data into ES with date from -18 hours earlier.

I'm not sure where the issue is, maybe in suricata or logstash?

I tried setting the system to UTC, and restarting everything.  Same issue.

Any suggestions?

**Comment by pevma** [comment]
On Thu, Nov 3, 2016 at 1:46 AM, aaronrryan <notifications@github.com> wrote:
> The data that is being pushed into ES has the wrong datetime.
> I have set the system to PDT timezone, but it inserting data into ES with
> date from -18 hours earlier.
>
> I'm not sure where the issue is, maybe in suricata?
>
> Should I set the system to GMT/UTC instead?

What time stamp is there on the actual suricata logs? Is it the expected one?
Where have you set the timezone?

>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.



-- 
Regards,
Peter Manev

**Comment by aaronrryan** [comment]
/var/log/suricata/eve.json is showing UTC time now.

I used "sudo  timedatectl set-timezone Etc/UTC" to set the localtime on the box.
Initially I set it to Pacific time, when I installed the system from the ISO, and the logs were login with -700 time (PDT), but showing up with the incorrect time in ES.

I think logstash hadnt caught up when I restarted everything, now that it has caught up the time is now correct.

So it seems the solution is to set the machines time to UTC and then"cat /dev/null > /var/log/suricata/eve.json" to clear out the old data and then purge ES data.

I'm really liking the SELKS 3.0, great job!

-Aaron
